<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="SerialBox5_fix_baudrate" name="serial_widget" native="true"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="GLWidget" name="gl_widget" native="true"/>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="sl_Y">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="sl_X">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSlider" name="sl_Z">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QSpinBox" name="sb_X"/>
               </item>
               <item row="1" column="2">
                <widget class="QSpinBox" name="sb_Y"/>
               </item>
               <item row="2" column="2">
                <widget class="QSpinBox" name="sb_Z"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="grapher_tab">
          <attribute name="title">
           <string>Grapher</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="GrapherBox" name="grapher_widget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sensors_tab">
          <attribute name="title">
           <string>Sensors</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="3" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>AccelMotion</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>AccelGravity</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Magnet</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Accel</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Y</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>LinAccel</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Gravity</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Result</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Z</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLCDNumber" name="lcd_accel_x"/>
                </item>
                <item row="1" column="2">
                 <widget class="QLCDNumber" name="lcd_accel_y"/>
                </item>
                <item row="1" column="3">
                 <widget class="QLCDNumber" name="lcd_accel_z"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLCDNumber" name="lcd_result_x"/>
                </item>
                <item row="2" column="2">
                 <widget class="QLCDNumber" name="lcd_result_y"/>
                </item>
                <item row="2" column="3">
                 <widget class="QLCDNumber" name="lcd_result_z"/>
                </item>
                <item row="3" column="1">
                 <widget class="QLCDNumber" name="lcd_accelmotion_x"/>
                </item>
                <item row="3" column="2">
                 <widget class="QLCDNumber" name="lcd_accelmotion_y"/>
                </item>
                <item row="3" column="3">
                 <widget class="QLCDNumber" name="lcd_accelmotion_z"/>
                </item>
                <item row="4" column="1">
                 <widget class="QLCDNumber" name="lcd_accelgravity_x"/>
                </item>
                <item row="4" column="2">
                 <widget class="QLCDNumber" name="lcd_accelgravity_y"/>
                </item>
                <item row="4" column="3">
                 <widget class="QLCDNumber" name="lcd_accelgravity_z"/>
                </item>
                <item row="5" column="1">
                 <widget class="QLCDNumber" name="lcd_linaccel_x"/>
                </item>
                <item row="5" column="2">
                 <widget class="QLCDNumber" name="lcd_linaccel_y"/>
                </item>
                <item row="5" column="3">
                 <widget class="QLCDNumber" name="lcd_linaccel_z"/>
                </item>
                <item row="6" column="1">
                 <widget class="QLCDNumber" name="lcd_gravity_x"/>
                </item>
                <item row="6" column="2">
                 <widget class="QLCDNumber" name="lcd_gravity_y"/>
                </item>
                <item row="6" column="3">
                 <widget class="QLCDNumber" name="lcd_gravity_z"/>
                </item>
                <item row="7" column="1">
                 <widget class="QLCDNumber" name="lcd_magnet_x"/>
                </item>
                <item row="7" column="2">
                 <widget class="QLCDNumber" name="lcd_magnet_y"/>
                </item>
                <item row="7" column="3">
                 <widget class="QLCDNumber" name="lcd_magnet_z"/>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GrapherBox</class>
   <extends>QWidget</extends>
   <header>grapherbox.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialBox5_fix_baudrate</class>
   <extends>QWidget</extends>
   <header>serialbox5_fix_baudrate.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
